dadi/lib/view/index.js

```diff
@@ -7,60 +7,54 @@ const path = require('path')
 const templateStore = require(path.join(__dirname, '/../templates/store'))
 const config = require(path.join(__dirname, '/../../../config.js'))
 
-const View = function (url, page, json) {
+const View = function (url, page) {
   this.url = url
   this.page = page
-  this.json = json
   this.data = {}
 
   this.templateName =
     this.page.hostKey +
     this.page.template.slice(0, this.page.template.indexOf('.'))
 
+  this.templateEngineSettings = this.page.settings.engine || {}
```

Comment:
this is likely a breaking change, is it documented as such?

Reply:
https://github.com/dadi/web/issues/255#issuecomment-373714581